### ISSUE TITLE
Document folder pausing

### ISSUE THREAD

**Issue by calmh** [syncthing/docs#326]
There is no mention of the mechanism of fields currently that I could find

**Comment by bean5** [comment]
@calmh This is pretty old. Is it still valid?

**Comment by marbens-arch** [comment]
> Is it still valid?

No. It was resolved in f0d0a05fa550d06063631cb9f210ee762352279b.
